### PR TITLE
Improved issues templates

### PR DESCRIPTION
**Pull request by george-gca**
Since a lot of issues opened are just questions that should be opened as so, and sometimes the issue template is just not followed, I decided to make opening new issues a little more "enforced", if not clear.

I based the new templates on the ones from [pytorch lightning](https://github.com/Lightning-AI/pytorch-lightning/issues/new/choose), which I think are pretty good. The visuals will only appear after the merge is processed, but you can have an idea of how it will look like by checking their templates and how they look.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    id: os
+    attributes:
+      label: What operating system are you using?
+      description: select all OSs where you have experienced this issue
+      multiple: true
+      options:
+        - Linux
+        - Mac
+        - Windows
+        - Not applicable
```
Comment (alshedivat) [inline .github/ISSUE_TEMPLATE/1_bug_report.yml:85]: what does it mean that OS "not applicable"?

File: .github/ISSUE_TEMPLATE/1_bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+  - type: dropdown
+    id: environment
+    attributes:
+      label: Where are you seeing the problem on?
+      description: select all environments where you have experienced this issue
+      multiple: true
+      options:
+        - "Running locally with Docker"
+        - "Running locally without Docker"
```
Comment (alshedivat) [inline .github/ISSUE_TEMPLATE/1_bug_report.yml:70]: if running locally w/o docker, it'd be good to ask about jekyll and ruby versions

**Review by alshedivat — APPROVED** [review]
LGTM, thanks for adding a much better template! left just a couple quick suggestions.

**Review by pourmand1376 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/1_bug_report.yml (hunk 1 above)
Comment (george-gca) [reply .github/ISSUE_TEMPLATE/1_bug_report.yml:85]: I put it there for the case when the error is when running on web, like after deploying. What do you think?

Comment (george-gca) [reply .github/ISSUE_TEMPLATE/1_bug_report.yml]: Changed text to explicit when it is not applicable.

**Pull request merged**